contracts/optimistic-ethereum/mockOVM/bridge/mockOVM_GenericCrossDomainMessenger.sol

```diff
@@ -40,7 +40,8 @@ contract mockOVM_GenericCrossDomainMessenger {
         public
     {
         xDomainMessageSender = _sender;
-        _target.call{gas: _gasLimit}(_message);
+        (bool success, ) = _target.call{gas: _gasLimit}(_message);
```

Comment:
If I'm not mistaken, this is a devex improvement disguised as a compiler fix but we'll allow it 😛

@smartcontracts I think you have the knowledge of this contract; seems fine to me but you know best.

Reply:
Yeah this is fine. Contract only gets used inside of our plugins to simulate L1<>L2 comms anyway.